rectangles/test/rectangle-test.js

```diff
@@ -0,0 +1,22 @@
+const assert = require('chai').assert
+const rectangles = require('../rectangle')
+
+const rectangle = { width: 3, height: 4 }
+
+describe('rectangle peroperty', function() {
```

Comment:
good work with your tests, i encourage you to ensure your method/variable names make sense to someone reading it `rectangles.area(rectangle)` isn't as clear and concise as it could be 👍